Source/Core/DolphinWX/Debugger/MemoryWindow.cpp

```diff
@@ -94,7 +94,13 @@ CMemoryWindow::CMemoryWindow(wxWindow* parent, wxWindowID id,
 	sizerBig->Add(sizerRight, 0, wxEXPAND | wxALL, 3);
 	sizerRight->Add(addrbox = new wxTextCtrl(this, IDM_MEM_ADDRBOX, ""));
 	sizerRight->Add(valbox = new wxTextCtrl(this, IDM_VALBOX, ""));
-	sizerRight->Add(new wxButton(this, IDM_SETVALBUTTON, _("Set &Value")));
+	sizerRight->Add(new wxButton(this, IDM_SETVALBUTTON,
+#ifdef __APPLE__
+		_("Set Value")
+#else
+		_("Set &Value")
+#endif
```

Comment:
I'd opt for just removing the accelerator. Having an accelerator there is very out of place.